docs/StardustDocs/topics/rename.md

```diff
@@ -8,3 +10,57 @@ df.rename { columns }.into { nameExpression }
 
 nameExpression = (DataColumn) -> String
 ```
+
+<!---FUN rename-->
+<tabs>
+<tab title="Properties">
+
+```kotlin
+df.rename { name }.into("fullName")
+```
+
+</tab>
+<tab title="Accessors">
+
+```kotlin
+val name by columnGroup()
+df.rename { name }.into("fullName")
+```
+
+</tab>
+<tab title="Strings">
+
+```kotlin
+df.rename("name").into("fullName")
+```
+
+</tab></tabs>
+<dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Modify.rename.html"/>
+<!---END-->
+
+
+<!---FUN renameExpression-->
+<tabs>
+<tab title="Properties">
+
+```kotlin
+
```

Comment:
is it ok to have here empty sections?

Reply:
indeed, looks like korro skipped these @koperagen

Reply:
My bad! Fixed it, thank you